priv/riak_kv.schema

```diff
@@ -121,7 +121,34 @@
 %% faster by doubling.  There are 1M segments in a standard tree overall.
 {mapping, "tictacaae_maxresults", "riak_kv.tictacaae_maxresults", [
   {datatype, integer},
-  {default, 256},
+  {default, 64}
+]}.
+
+%% @doc Max number of repair loops per exchange
+%% Each exchange will attempt a repair of tictacaae_maxresults, and will
+%% analyse those repairs to see if there exists the potential for more repairs.
+%% If there exists this potential, then repair loops will be run, but in these
+%% repair loops a more efficient repair will be run:
+%% - with a bucket, key_range or last_modified range (at least) to restrict the
+%% scope of the fetch_clocks queries
+%% - without rebuilding the segments in the AAE hash tree
+%% - without checking for presence of th eobject in the journal
+%% Problems related to invalid hash trees will need to be detected through the
+%% initial loop of the exchange, not repair loops
+{mapping, "tictacaae_repairloops", "riak_kv.tictacaae_repairloops", [
+  {datatype, integer},
+  {default, 4},
+  hidden
+]}.
+
+%% @doc Multiplier when following an exchange with a range-limited one
+%% After eache xchange where sufficient deltas are discovered there will be
+%% `tictacaae_repairloops` number of range-limited queries (assuming
+%% sufficient results continue to be found).  Each of these may have the
+%% the number of max results boosted by this integer factor.
+{mapping, "tictacaae_rangeboost", "riak_kv.tictacaae_rangeboost", [
+  {datatype, integer},
+  {default, 2},
```

Comment:
Is this now 4 in line with `256 div 64` is 4? If that is the case, then I wonder if it should be a configurable constant or whether it is a derived constant. I did not really get this from the above explanation. Tried to correlated it what was written in the commit message:

`These loops will have the tictacaae_maxresults boosted by a tictacaae_rangeboost integer (default 2 i.e. go to 256).`

Reply:
Clarified comments, included examples